modules/core/shared/src/main/scala/lucuma/core/model/sequence/flamingos2/Flamingos2DynamicConfig.scala

```diff
@@ -6,29 +6,32 @@ package lucuma.core.model.sequence.flamingos2
 import cats.Eq
 import cats.syntax.all.*
 import lucuma.core.enums.*
+import lucuma.core.math.Wavelength
 import lucuma.core.util.TimeSpan
 import monocle.Focus
 import monocle.Lens
 
 case class Flamingos2DynamicConfig(
-  exposure:       TimeSpan,
-  disperser:      Option[Flamingos2Disperser],
-  filter:         Flamingos2Filter,
-  readMode:       Flamingos2ReadMode,
-  lyot:           Flamingos2LyotWheel,
-  fpu:            Flamingos2FpuMask,
-  readoutMode:    Option[Flamingos2ReadoutMode], // engineering
-  reads:          Option[Flamingos2Reads], // engineering
-) {
-  val isImaging: Boolean       = fpu.isImaging
-  val isMOS: Boolean           = fpu.isMOS
-  val isLongSlit: Boolean      = fpu.isLongSlit
-  val decker: Option[Flamingos2Decker] = fpu.fold(none, _.value.decker.some, _ => none)
-}
+  exposure:    TimeSpan,
+  disperser:   Option[Flamingos2Disperser],
+  filter:      Flamingos2Filter,
+  readMode:    Flamingos2ReadMode,
+  lyotWheel:   Flamingos2LyotWheel,
+  fpu:         Flamingos2FpuMask,
+  decker:      Flamingos2Decker,
+  readoutMode: Flamingos2ReadoutMode,
+  reads:       Flamingos2Reads
+):
+  val isImaging: Boolean  = fpu.isImaging
+  val isMOS: Boolean      = fpu.isMOS
+  val isLongSlit: Boolean = fpu.isLongSlit
+
+  val centralWavelength: Wavelength =
```

Comment:
In Explore on the configuration panel we are just using the filter wavelength for the central wavelength. (We use it for calculating the λ interval.) I used this at Andy's suggestion. I wonder if it should use the same method as here instead?

Reply:
I took this from the `ocs` project where we have:

```java
    /**
     * Determines the observing wavelength to use for the given combination of
     * disperser and filter.  Implements the algorithm specified in SCI-0206.
     */
    public static Option<Double> getObservingWavelength(Disperser d, Filter f) {
        // When in spectroscopy mode, if the filter is OPEN, use the grism
        // wavelength.
        if ((d != Disperser.NONE) && (f == Filter.OPEN)) return d.getWavelength();

        // Otherwise, use the filter wavelength.
        return f.getWavelength();
    }
```

Note `Filter.OPEN` is obsolete.  Now we get into the difference between "central wavelength" and "observing wavelength" which admittedly baffles me.

Reply:
Yes. There are too many wavelengths.

Reply:
Maybe since the filter is never `OPEN` now (since it doesn't exist), I should just use the filter wavelength.  Is this actually the "observing wavelength" or the "central wavelength" or neither or both?

Reply:
I just asked Andy what to use for the central wavelength, and he said he "would be inclined to use the filter wavelength" - or something very similar. Maybe we should tag him here? You could merge as is and update if needed.

Reply:
Or, I could update explore if needed.

Reply:
> Or, I could update explore if needed.

I'll switch to filter since `OPEN` doesn't exist.

Reply:
I think we should be using the filter wavelength.
imaging will *always* use a filter.
F2 will *always* use a filter (imaging and spectroscopy).

Reply:
We support n filters right? Does this dynamic config apply to only one part of a sequence with multiple filters?

Reply:
The dynamic config is how the instrument is configured for one F2 dataset.  If there are multiple filters in the observing mode, they would be used in different steps.